public/window/ready.js

```diff
@@ -0,0 +1,25 @@
+const path = require('path');
+const fs = require('fs');
+
+function handleCommandLine(mainWindow) {
+  let directory;
+  if (path.isAbsolute(process.argv[1])) {
+    directory = process.argv[1];
+  } else {
+    directory = path.join(process.cwd(), process.argv[1]);
+  }
+  if (fs.lstatSync(directory).isDirectory()) {
+    if (mainWindow) {
```

Comment:
This could be one-lined with the parent `if` condition using:`... isDirectory() && mainWindow) { ... }